src/project-1.js

```diff
@@ -11,6 +12,10 @@ const subtractFive = (num) => {
 };
 
 const areSameLength = (str1, str2) => {
```

Comment:
You could just return what's inside the if statement, in the format
`const areSameLength = (str1, str2) => str1.length === str2.length;`